sw/device/lib/dif/dif_spi_device.h

```diff
@@ -23,54 +23,108 @@ extern "C" {
 #endif  // __cplusplus
 
 /**
- * Represents a signal edge type.
+ * A signal edge type: positive or negative.
  */
 typedef enum dif_spi_device_edge {
+  /**
+   * Represents a positive edge (i.e., from low to high).
+   */
   kDifSpiDeviceEdgePositive,
+  /**
+   * Represents a negative edge (i.e., from high to low).
+   */
   kDifSpiDeviceEdgeNegative,
 } dif_spi_device_edge_t;
 
 /**
- * Represents a bit ordering.
+ * A bit ordering within a byte.
  */
 typedef enum dif_spi_device_bit_order {
+  /**
+   * Represents the most-significant-bit to least-significant-bit order.
+   */
   kDifSpiDeviceBitOrderMsbToLsb,
+  /**
+   * Represents the least-significant-bit to most-significant-bit order.
+   */
   kDifSpiDeviceBitOrderLsbToMsb,
 } dif_spi_device_bit_order_t;
 
 /**
- * Represents the enablement state of a particular SPI IRQ.
+ * A toggle state: enabled, or disabled.
+ *
+ * This enum may be used instead of a `bool` when describing an enabled/disabled
+ * state.
```

Comment:
This probably belongs to the template but not here, especially considering `...Enabled` is `0`.

Reply:
I think I agree with this, but I think we should fix this wording in the template. Could you file an issue?